svc/GridExportService.js

```diff
@@ -106,19 +106,20 @@ export class GridExportService {
             includeAll = toExport.includes('ALL'),
             includeViz = toExport.includes('VISIBLE');
 
-        return gridModel
-            .getLeafColumns()
-            .filter(col => {
-                const {colId, excludeFromExport} = col;
-                return (
-                    !excludeFromExport &&
-                    (
-                        includeAll ||
-                        toExport.includes(colId) ||
-                        (includeViz && gridModel.isColumnVisible(colId))
-                    )
-                );
-            });
+        return sortBy(gridModel.getLeafColumns(), ({colId}) => {
+            const match = gridModel.columnState.find(it => it.colId === colId);
+            return withDefault(match?._sortOrder, 0);
```

Comment:
Should we be promoting `_sortOrder` to a public, documented field on our `ColumnState` typedef?